cdi/src/test/java/org/togglz/cdi/container/ManagedFeatureManagerProvider.java

```diff
@@ -1,17 +1,17 @@
 package org.togglz.cdi.container;
 
-import javax.enterprise.context.ApplicationScoped;
-import javax.enterprise.inject.Produces;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.enterprise.inject.Produces;
 import org.togglz.cdi.Features;
 import org.togglz.core.manager.FeatureManager;
 import org.togglz.core.manager.FeatureManagerBuilder;
 import org.togglz.core.repository.mem.InMemoryStateRepository;
 import org.togglz.core.user.NoOpUserProvider;
 
+@ApplicationScoped
```

Comment:
why do we need to put it on the class instead of just keeping it on the produce() method?

Reply:
One of Marco's commits says:
 Fix CDI module, the @ApplicationScoped annotation has to be at class level to be discovered by new default bean-discovery-mode annotated in CDI 4.0
So, looks like yes, it should be exactly on the class level.

Reply:
Okay. Interesting. Then this is fine.